src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -1756,4 +1780,64 @@ TEST_CASE(
   CHECK(num_sent_requests == doctest::Approx(50).epsilon(0.1));
 }
 
+TEST_CASE("request rate manager - reconfigure threads")
```

Comment:
look for anywhere you had "reconfigure" and update it to match the new name (configure)